### ISSUE TITLE
POP_JUMP_FORWARD_IF_FALSE Unsupported opcode

### ISSUE THREAD

**Issue by Remi-Avec-Un-I** [zrax/pycdc#332]
Hello, I'm having this error: POP_JUMP_FORWARD_IF_FALSE
from a 3.11 python compiled script.

This is an exemple of how to reproduce this:
```
import dis

def foo(x):
    if x > 0:
        print("x is positive")
    else:
        print("x is not positive")

dis.dis(foo)
```

I don't know if it helps, tell me if you need more information

**Comment by Remi-Avec-Un-I** [comment]
@TheHellTower

**Comment by TheHellTower** [comment]
> import dis
> 
> def foo(x):
>     if x > 0:
>         print("x is positive")
>     else:
>         print("x is not positive")
> 
> dis.dis(foo)

Hello,

Thank you for your contribution to make the project better, I will check this when I will have free time, if you get other opcodes that are not supported along with a example to reproduce them, don't hesitate to open more issues it would be really helpful.

Regards.

**Comment by Remi-Avec-Un-I** [comment]
No problem, I'ld like to help more, if you can tell how to modify or add opcode directly form source code i could try.

**Comment by TheHellTower** [comment]
> No problem, I'ld like to help more, if you can tell how to modify or add opcode directly form source code i could try.

I'm not that much in python programming or even searching codes to see if a opcode is present in there when compiled so I can't help with that I only can help pycdc to go forward.

**Comment by TheHellTower** [comment]
> Hello, I'm having this error: POP_JUMP_FORWARD_IF_FALSE from a 3.11 python compiled script.
> 
> This is an exemple of how to reproduce this:
> 
> ```
> import dis
> 
> def foo(x):
>     if x > 0:
>         print("x is positive")
>     else:
>         print("x is not positive")
> 
> dis.dis(foo)
> ```
> 
> I don't know if it helps, tell me if you need more information

I will verify this in the next week, I might have some free time.

**Comment by TheHellTower** [comment]
> Hello, I'm having this error: POP_JUMP_FORWARD_IF_FALSE from a 3.11 python compiled script.
> 
> This is an exemple of how to reproduce this:
> 
> ```
> import dis
> 
> def foo(x):
>     if x > 0:
>         print("x is positive")
>     else:
>         print("x is not positive")
> 
> dis.dis(foo)
> ```
> 
> I don't know if it helps, tell me if you need more information

Sorry for being gone I had some things to do(even if I didn't solve everything) I will try to take a look at it soon. But I don't have a date in mind about the update push date.

**Comment by zrax** [comment]
Resolved with #365
